install-pcf/aws/tasks/prepare-aws/task.sh

```diff
@@ -8,14 +8,14 @@ OPSMAN_ALLOW_SSH=0
 OPSMAN_ALLOW_SSH_CIDR_LIST='["0.0.0.0/32"]'
 if [[ -n "${OPSMAN_ALLOW_SSH_CIDR_RANGES// }" ]]; then
   OPSMAN_ALLOW_SSH=1
-  OPSMAN_ALLOW_SSH_CIDR_LIST='["'${OPSMAN_ALLOW_SSH_CIDR_RANGES/\,/\"\,\"}'"]'
+  OPSMAN_ALLOW_SSH_CIDR_LIST='["'${OPSMAN_ALLOW_SSH_CIDR_RANGES//\,/\"\,\"}'"]'
```

Comment:
This might be better done via `sed`:

```
echo "some,values,here" | sed -e 's/\([^,]\+\)/"\1"/g'
"some","values","here"
```

And also not inlined.

Thoughts?